environment/src/main/java/com/sequenceiq/environment/environment/v1/EnvironmentController.java

```diff
@@ -141,7 +143,7 @@ public EnvironmentCrnResponse getCrnByName(@ResourceName String environmentName)
 
     @Override
     @CheckPermissionByResourceCrn(action = AuthorizationResourceAction.DESCRIBE_ENVIRONMENT)
-    public DetailedEnvironmentResponse getByCrn(@ResourceCrn @TenantAwareParam String crn) {
+    public DetailedEnvironmentResponse getByCrn(@ValidCrn(resource = CrnResourceDescriptor.ENVIRONMENT) @ResourceCrn @TenantAwareParam String crn) {
```

Comment:
static import ? we used it in EnvironmentEndpoint.java

Reply:
because there were conflicts between static imports (CrnResourceDescriptor.ENVIRONMENT vs CredentialType.ENVIRONMENT) i decided to not use static import in case of CrnResourceDescriptor